code/game/objects/items/cards_ids.dm

```diff
@@ -871,6 +920,52 @@
 		return insert_money(interacting_with, user) ? ITEM_INTERACT_SUCCESS : ITEM_INTERACT_BLOCKING
 	return NONE
 
+/obj/item/card/id/item_ctrl_click(mob/user)
```

Comment:
Should only allow this if the id card is being held (so you can still drag them if you feel so inclined)